test/banking_test.lua

```diff
@@ -88,8 +89,16 @@ function g.test_banking()
 
     srv.net_box:call('transfer_money', {1, 2, 20})
 
+    http_client.get(string.format('localhost:%s/customer_add?customer_id=%s&fullname=%s',
+                                              srv.http_port, '3', 'Artur'))
+    http_client.get(string.format('localhost:%s/account_add?customer_id=%s&account_id=%s&name=%s',
+                                              srv.http_port, '3', '3', 'default'))
```

Comment:
It could be more concise if you used json body instead of `query_param`.

There're lots of good kv home task examples, e.g. https://github.com/vanyarock01/remote-web-storage/